tests/Unit/Evolution/Systems/ScalarWave/Test_Equations.cpp

```diff
@@ -51,22 +51,25 @@ void check_du_dt(const size_t npts, const double time) {
   }();
 
   auto box = db::create<db::AddSimpleTags<
-      Tags::dt<ScalarWave::Pi>, Tags::dt<ScalarWave::Phi<Dim>>,
-      Tags::dt<ScalarWave::Psi>, ScalarWave::Pi,
+      ScalarWave::Tags::ConstraintGamma2, Tags::dt<ScalarWave::Pi>,
+      Tags::dt<ScalarWave::Phi<Dim>>, Tags::dt<ScalarWave::Psi>, ScalarWave::Pi,
+      ScalarWave::Phi<Dim>,
       Tags::deriv<ScalarWave::Pi, tmpl::size_t<Dim>, Frame::Inertial>,
+      Tags::deriv<ScalarWave::Psi, tmpl::size_t<Dim>, Frame::Inertial>,
       Tags::deriv<ScalarWave::Phi<Dim>, tmpl::size_t<Dim>, Frame::Inertial>>>(
       Scalar<DataVector>(pow<Dim>(npts), 0.0),
+      Scalar<DataVector>(pow<Dim>(npts), 0.0),
       tnsr::i<DataVector, Dim, Frame::Inertial>(pow<Dim>(npts), 0.0),
       Scalar<DataVector>(pow<Dim>(npts), 0.0),
       Scalar<DataVector>(-1.0 * solution.dpsi_dt(x, time).get()),
-      [&x, &time, &solution ]() noexcept {
+      solution.dpsi_dx(x, time), [&x, &time, &solution ]() noexcept {
         auto dpi_dx = solution.d2psi_dtdx(x, time);
         for (size_t i = 0; i < Dim; ++i) {
           dpi_dx.get(i) *= -1.0;
         }
         return dpi_dx;
       }(),
-      [&npts, &x, &time, &solution ]() noexcept {
+      solution.dpsi_dx(x, time), [&npts, &x, &time, &solution ]() noexcept {
         tnsr::ij<DataVector, Dim, Frame::Inertial> d2psi_dxdx{pow<Dim>(npts),
                                                               0.0};
         const tnsr::ii<DataVector, Dim, Frame::Inertial> ddpsi_soln =
```

Comment:
It appears that you are testing the dudt function with a gamma2 of `0.0`, which doesn't verify that the change in this PR behaves as expected. You should use a nonzero gamma2 and verify that the damping parameter is appropriately reflected in the DuDt mutation.

Reply:
Probably once with the constraint satisfied and once without

Reply:
Included similar test with non-trivial values for `gamma2`, once with constraint satisfied and once without